mlir/include/mlir/Dialect/OpenMP/OpenMPClauses.td

```diff
@@ -503,6 +503,61 @@ class OpenMP_IsDevicePtrClauseSkip<
 
 def OpenMP_IsDevicePtrClause : OpenMP_IsDevicePtrClauseSkip<>;
 
+//===----------------------------------------------------------------------===//
+// V5.2: [5.4.7] `inclusive` clause
+//===----------------------------------------------------------------------===//
+
+class OpenMP_InclusiveClauseSkip<
```

Comment:
Nit: Move definitions for these new clauses to keep alphabetical sorting in the file.